src/main/java/racingcar/controller/Race.java

```diff
@@ -0,0 +1,33 @@
+package racingcar.controller;
+
+import racingcar.model.Cars;
+import racingcar.model.RaceResult;
+import racingcar.model.RandomNumberStrategy;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
+
+public class Race {
+
+    private final InputView inputView;
+    private final ResultView resultView;
+
+    public Race(InputView inputView, ResultView resultView) {
+        this.inputView = inputView;
+        this.resultView = resultView;
+    }
+
+    public void start() {
+        resultView.printCarAmount();
+        int carAmount = inputView.inputNumber();
+        resultView.printTryNumber();
+        int tryNumber = inputView.inputNumber();
+
+        Cars cars = new Cars(carAmount);
+
+        RaceResult raceResult = cars.tryMove(tryNumber, new RandomNumberStrategy());
+        resultView.printResultNotice();
+        resultView.printTryResult(raceResult);
+    }
+
+
+}
```

Comment:
```suggestion
    }
}
```
여기서의 개행은 굳이 없어도 될거 같아요~

Reply:
아 꼼꼼하지 못했네요 감사합니다 ㅎㅎ